Cargo.toml

```
@@ -21,7 +21,7 @@ include = [
 path = "bindings/rust/lib.rs"
 
 [dependencies]
-tree-sitter = ">= 0.21, < 0.23"
+tree-sitter = "0.23"
```

The changes in tree-sitter 0.24 are actually backwards compatible, meaning that tree-sitter 0.24 works with grammars written for tree-sitter 0.23. So I think it would be very helpful to relax the dependency constraint and let folks use the parser with newer versions of tree-sitter:
```suggestion
tree-sitter = ">=0.23"
```